### PR TITLE
BFX-405 file upload

### PR DESCRIPTION
**Pull request by sosazuwaVIA**
## Changes

Add documentation to upload data:
* Via AWS CLI
* Via xCloud
* Via Cyber Duck
* Add Relevant images

## Reviewer Notes

* Preview on the site: http://127.0.0.1:8000/ViaFoundry/data/data-guide/
  * Replace `127.0.0.1:8000` with wherever you're serving
* I intend to remove the CyberDuck pictures in favor of a Youtube link. I just want to get this PR in your hands to get thoughts flowing in

### CODE REVIEW

File: docs/ViaFoundry/data/data-guide.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Getting Your Data on Via Foundry
+
+Welcome to the "Data Guide" section! This is your go-to resource for learning how to efficiently connect and manage your data with Via Foundry. Whether you're integrating data from external sources or uploading local datasets, we've got you covered. In this section, we'll walk you through the various features and methods Via Foundry offers for seamless data integration.
+
+## Connect to your Data in the Cloud
+
+### Connect your storage with Foundry xCloud
+
+Foundry [xCloud](https://www.viascientific.com/products/) enables your cloud storage available in Via Foundry. [xCloud](https://www.viascientific.com/products/) allows organizations and labs to integrate Foundry with their existing cloud infrastructure. Reach out to support to get started, `support@viascientific.com`.
```
Comment (sosazuwaVIA) [inline docs/ViaFoundry/data/data-guide.md]: From @tony-vs 

Move to a different section. Make it clear that this isn't "Connecting" storage. 

xCloud places Foundry in your environment

File: docs/ViaFoundry/data/data-guide.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+### Connect your Amazon S3 Bucket directly to Foundry
+
+For users seeking greater control of data management we allow connect your private S3 bucket to Via Foundry [vCloud](https://www.viascientific.com/products/) product. Reach out to support to get started, `support@viascientific.com`.
+
+## Upload to our bucket from the CLI using `aws cli`
```
Comment (sosazuwaVIA) [inline docs/ViaFoundry/data/data-guide.md]: From @tony-vs 

Make sure it's clear region has to be AWS us-east 1

File: docs/ViaFoundry/data/data-guide.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### Prerequisites:
+
+Install AWS CLI following the instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+### Step-by-Step Instructions:
```
Comment (sosazuwaVIA) [inline docs/ViaFoundry/data/data-guide.md]: From @tony-vs 

Update to make "Organization of data in bucket" clear and give example

File: docs/ViaFoundry/data/data-guide.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+     ```
+
+### Example:
+If you have a file `data.csv` and a bucket name `via-foundry-bucket`, your commands will look like this:
+```bash
+aws s3 cp data.csv s3://via-foundry-bucket/
+aws s3 ls s3://via-foundry-bucket/
+```
+
+## Upload with Cyber Ducker
```
Comment (sosazuwaVIA) [inline docs/ViaFoundry/data/data-guide.md]: From @tony-vs

Update to make "Organization of data in bucket" clear and give example

File: docs/ViaFoundry/data/data-guide.md (hunk 2 above)
Comment (sosazuwaVIA) [inline docs/ViaFoundry/data/data-guide.md]: From discussion with @tony-vs 

Showcase connection with AWS. Adding Amazon Keys.

File: docs/ViaFoundry/data/data-guide.md (hunk 4 above)
Comment (sosazuwaVIA) [inline docs/ViaFoundry/data/data-guide.md]: From discussion with @tony-vs 

Make sure it's clear we don't advocate for Cyber Duck. They can use other tools (recommend other tools). We're not liable for how they use Cyberduck

Comment (sosazuwaVIA) [inline docs/ViaFoundry/data/data-guide.md]: From discussion with @tony-vs 

They need to add Amazon Keys.

**Review by tony-vs — COMMENTED** [review]
LGTM, if those images can be larger frin the focused area that'd be more helpful IMO.

**Review by tony-vs — APPROVED** [review]
looks better! thanks

**Pull request merged**